packages/core/src/browser/keybinding.spec.ts

```diff
@@ -157,10 +157,10 @@ describe('keybindings', () => {
 
         const bindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND2.id);
         if (bindings) {
-            expect(bindings.length).to.be.equal(2);
```

Comment:
@marechal-p @svenefftinge Could you help me to figure out why this test is failing? I cannot get what is testing.

Reply:
I don't understand either... I think before, registering something in the user scope would shadow anything in the default scope. But looking at the way it was before your change, I am confused.

Reply:
Same here, the test doesn't make sense to me